app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt

```diff
@@ -78,6 +79,7 @@ class AudioRecording {
                     val byteArray = readBuffer.sliceArray(0 until actualRead)
                     val audioChunk = AudioChunk(byteArray, actualRead, System.currentTimeMillis())
                     emit(audioChunk)
+                    delay(10)
```

Comment:
what's the delay for?

Reply:
Without it there is too much work on the main thread and the app crashes, the delay allows for enough time to transmit the chunk before recording and storing another